Remove UriFix, needed in net45 but now oxbow code

louis-z opened the pull request.

louis-z committed:
Remove UriFix, needed in net45 but now oxbow code

meziantou commented:
How did you validate the change as the tests currently only validate net5? (manual?)

Maybe we should multi-target net48 to be sure NGitLab works on .NET Framework.

louis-z commented:
Before removing the UriFix test, I
- modified the project to target `net472` (which meant commenting out a bunch of `net5.0`-specific stuff...)
- modified the test to instantiate a `Uri` directly

The test ran fine.
